src/libpmemfile-posix/dir.c

```diff
@@ -211,18 +211,17 @@ vinode_replace_debug_path_locked(PMEMfilepool *pfp,
  */
 void
 vinode_add_dirent(PMEMfilepool *pfp,
```

Comment:
"posix: remove inode_ref_new ... This means that sometimes we have to pass around inodes instead
of vinodes."
So shouldn't this function be called `inode_add_dirent` now?